src/lib/message-queue-v2/QueueManager.js

```diff
@@ -5,6 +5,11 @@ const bull = require('bullmq');
 const STATUS_COMPLETED = 'completed';
 const STATUS_FAILED = 'failed';
 
+const jobDefaults = {
+  removeOnComplete: true,
+  removeOnFail: 100
```

Comment:
I think 100 might be low. May I suggest 500ish?

Reply:
Done